1-A/bqelibari/bqelibari.txt

```diff
@@ -0,0 +1,153 @@
+
+	
+
+Ein Prozess bekommt seinen eigenen Adressraum und weitere Betriebssystemmittel/Ressourcen zugeordnet.
+	-Prozesse sind gegeneinander abgeschrimt.
+	-Prozesse können nur auf eigene Adressräume und Betriebssystemmittel zugreifen.
+Im Gegensatz ein Programm:
+	Statische verfahrensvorschrift für eine Verarbeitung auf einm Rechnersystem.
+	
+A Process is represented by a da datastructure called Process Control Block (PCB). The PCB contains many informations regarding its Process.
+e.g. its hardware-context, its current processor-state
+	
+Der Prozess_Scheduler:
+	-Steuerprogramm des Bertiebssystems
+		-Regelt die zeitliche Ausführung mehrerer Prozesse in Bertriebsystemen.
+		-Funktioniert wie eine Warteschlange. 
+		-(Soll in der Lage sein Echtzeit-Systeme zu händeln.)
+		-Kann mehrere Prozesse vor Beendung hin und her switchen, um das gleichzeitige Abl-
+		laufen mehrerer Prozesse zu simulieren.(Dabei kann nur ein Prozess gleichzeitg
+		verarbeitet werden.
+
+	-Grobe Unterteilung in:
+		-unterbrechende
+		-nicht unterbrechende Schedulingstrategien.
+
+	-Unterbrechende Scheduler:
+		-teilen dem CPU von vorneherein nur für eine bestimmte Zeitspanne dem Prozess zu und
+		entziehen Ihn dem Prozess wieder.
+
+	-Nicht unterbrechende Scheduler:
+		Lassen den CPU solange für den Prozess laufen, bis der Prozess den CPU wieder freigibt.
+		
+		
+Init-Prozess:
+	Ist bei UNIX-Systemen der allererste Prozess mit Prozess-ID 1 (PID1), unter windows gibt es keinen vergleichbaren prozess.
+	dieser startet alle anderen Prozesse. Welche das sind und in welcher reihenfolge diese gestartet werden,
+	ist in etc/inittab festgelegt.
+	ein kill init kommt dem Herunterfahren des Systems gleich, da mit dessen Beendigung alle anderen Prozesse beendet werden.
+	Diese anderen Prozesse werden beendet da sie Kind-Prozesse des Init-Prozesses sind.
+	Als aller erstes wird vor dem Init-Prozess jedoch einen POST (Power-On-Self-Test) vom BIOS durchgeführt,
+	bei diesem wird:
+		-Funktion der CPU
+		-Funktion der RAM
+		-Funktion der Cache
+		-Funktion der I/O-Software
+	geprüft. Sollte der POST fehlschlafǵen, ertönt ein signalton und der start des Computers wird abgebrochen.
+	
+
+Shell:
+	Software die das Interagieren mit dem OS möglich macht. 
+	Kernel = Kern des OS, Shell = Hülle, für den benutzer Sichtbarer teil des OS.
+	The visible part of the OS discribes the Console, the most fundamental option for HUmans to interact with an OS.
+	The shell is so powerful that there are basically no limitations, if you have sufficient permissions, the shell allows you to delete
+	the whole file system.
+	A login shell is the first process that executes under your user ID when you log in for an interactive session. 
+	Login shells typically read a file that do things like setting environment variables, whereas the virtual shell is just another name
+	for the regular shell. This was done in order to confuse ppl who want to start working with linux.
```

Comment:
Not quite right, but very amusing!
You should take a look at: [What is the difference between Terminal, Shell and Command Line?](https://askubuntu.com/questions/506510/what-is-the-difference-between-terminal-console-shell-and-command-line)